src/main.h

```diff
@@ -38,7 +38,7 @@ gboolean main_is_realized(void);
 
 typedef struct
 {
-	gboolean	new_instance;
+	gint		new_instance;
```

Comment:
> make it an enum with sensible names.

I just added a commit for this.

https://github.com/geany/geany/pull/1128/commits/9c0d06ae3de47a6d6adff0eab9bcebcd491f58cb